scm/driver/stash/pr.go

```diff
@@ -230,6 +230,16 @@ func convertPullRequest(from *pr) *scm.PullRequest {
 		Merged:  from.State == "MERGED",
 		Created: time.Unix(from.CreatedDate/1000, 0),
 		Updated: time.Unix(from.UpdatedDate/1000, 0),
+		Head: scm.Reference{
+			Name: from.FromRef.DisplayID,
+			Path: from.FromRef.ID,
+			Sha:  from.FromRef.LatestCommit,
+		},
+		Base: scm.Reference{
+			Name: from.ToRef.DisplayID,
+			Path: from.ToRef.ID,
+			Sha:  from.ToRef.LatestCommit,
```

Comment:
I think these should be correct. However, I didn't find use for anything else except `.Base.Sha` in Drone server

https://github.com/harness/drone/blob/master/service/hook/parser/parse.go#L275

So, should this addition be omitted from the PR?